### ISSUE TITLE
DHARMA and dharmalekha.info beyond ERC funding

### ISSUE THREAD

**Issue by arlogriffiths** [erc-dharma/project-documentation#287]
I would like to conceive of dharmalekha.info as a long-term platform, only connected with ERC-project DHARMA in its initial phase. If we all agree on this, then it has some consequences for how @michaelnmmeyer continues to develop dharmalekha.info. 

1. I would like the ERC to be mentioned only in specific acknowledgements of our funding sources, but not in places that will be displayed in the long run. Indeed, I already regretted way back ion 2019 that Axelle included "ERC" in setting up names for our Zotero Group and Github organisation way back in 2019 — @michaelnmmeyer: I hope we can get rid of these letters in due course, as we continue to use the github organisation and the zotero group beyond the end of the project.

2. At screen-bottom, we currently have an indication "© ERC-DHARMA Project, 2019-2024", underlyingly
`© [ERC-DHARMA Project](https://dharma.hypotheses.org/), 2019-2024`. Since the pages at https://dharma.hypotheses.org have never really come alive, and we may never have the time to dress them up to last as long as the sun and the moon, it may be better not to refer to them too often and too prominently on dharmalekha.info.

3. Moreover, it has never been correct that ERC-project DHARMA ends in 2024 (it was 2025 and has become 2026), and these statements seem contradictory to the © claims made in the xml files for individual inscriptions. Anyhow, at screen-bottom, I suggest we can replace "© ERC-DHARMA Project, 2019-2024" either by nothing at all, or if we need something then a phrase like "dharmalekha.info is the publication platform of the DHARMA consortium" — this could then link to a page where the past, present and future of the DHARMA consortium are explained with due mention of ERC funding but without suggesting limitation to the ERC project.

**Comment by manufrancis** [comment]
I agree on dharmalekha.info as a long-term platform, only connected with ERC-project DHARMA in its initial phase. 

As for Arlo's suggestions.

1. Why not? If easy and smooth operation.
In the same vein, what about "DHARMA" in
`<idno type="filename">DHARMA_INSTamilNadu00200</idno>`

2. and 3.
Fine with me.

@arlogriffiths 
As for the domain name, which was acquired by EFEO through Vincent Paillusson (who has left EFEO, if am right), you should make sure that we can renew our acquisition. I do not know how it works. I guess some email address receives an alert before expiration.

**Comment by arlogriffiths** [comment]
As for Manu's response to my point 1: indeed, I have been feeling that URL's like https://dharmalekha.info/texts/DHARMA_INSCIC00041 are annoyingly redundant (dharma/DHARMA) and uglier than they need to be with mix of upper and lower case. As I recall, when we made the FNC, it was considered essential to have a prefix for the ERC project. If that is still deemed essential, then it may nevertheless be possible for @michaelnmmeyer to suppress the prefix DHARMA_ in generating URLs. He may please give us a recommendation on what is technically possible and what seems advisable in terms of best practices.

Among the following three, the first is what we have now and the last would be the maximal change:

- https://dharmalekha.info/texts/DHARMA_INSCIC00041
- https://dharmalekha.info/texts/INSCIC00041
- https://dharmalekha.info/texts/inscic00041

Actually, the middle option looks fine to me, so pending Michaël's recommendations, that might have my preference.

@manufrancis : I will discuss this point with Michaël in due course. I don't remember exactly what we did with Vincent P. but he still answers emails and I am sure he'd be happy to refresh my memory. Anyhow, I am rather optimistic that EFEO would be happy to pay keep paying for the domain name even in the longer term.

**Comment by michaelnmmeyer** [comment]
@arlogriffiths 

1. It is possible to rename the organization both on github and on zotero. This will require people to update their setup, though.
2. OK.
3. OK, I will add an extra page.

@manufrancis @arlogriffiths 

Currently, the `DHARMA_` prefix is used to distinguish files that are supposed to appear in the database from other files people might store into their repositories. If we can agree on a convention that would allow the same distinction to be made (for instance, having a "dharma" folder that contains all the files), I am fine with this.

In any case, I can systematically remove the `DHARMA_` prefix in URIs and in identifiers. Having case-insensitive identifiers might cause issues, though.

**Comment by arlogriffiths** [comment]
@michaelnmmeyer 
1. can you explain what you mean by people updating their set up?
2. let us know when this is done
3. same

As for the DHARMA_ prefix, I have no real objection to keeping it in our file names: the use of the prefix to distinguish between files to be processed and files not to be processed is clearly of long-term relevance. If I understand correctly, file names and identifiers are essentially the same thing (see issue #266) and I have no problem with keeping these case-sensitive. So I'd simply wish us to consider the exclusion of prefix DHARMA_ from URIs.

**Comment by danbalogh** [comment]
I have no strong opinions about Arlo's original three points; I essentially agree but feel that giving more credit to the ERC is better than giving less. It costs us nothing and may win goodwill in the long run. 

On filenames and URIs, I think that now that the FNC and our file and folder structure are in place, it would be better not to change anything. We had better keep IDs case sensitive; this is not so evident for repositories like CIC, but we also have repositories like DaksinaKosala etc, where upper/lower case also conveys some human-readable information. I also think that it may be best if the permanent URI of a resource would remain https://dharmalekha.info/texts/DHARMA_INSCIC00041, and if we want to allow https://dharmalekha.info/texts/INSCIC00041, this sort of URL be set up as an alias that points to the above.

**Comment by danbalogh** [comment]
It seems to me that `<idno type="filename">` has been inherited from the generic EpiDoc template, but I find no explanation in the EpiDoc guidelines about why this type should be used, and I do now know why we have opted to type our idnos as filenames but omit the extension. Looking at some examples of idno type usage in TEI, I think it would be perfectly ok to use `<idno type="dharma">` instead. But I'm OK with any other token we decide on for type, and also OK with adding the extension here.
